### PR TITLE
pandas-compat: add recipe

### PR DESCRIPTION
**Pull request by jreback**

### CODE REVIEW

File: recipes/pandas-compat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+    - pandas
+
+test:
+  imports:
+    - pandas_compat
+
+about:
+  home: https://github.com/pandas-dev/pandas-compat
+  license: BSD 3-clause
```
Comment (synapticarbors) [inline recipes/pandas-compat/meta.yaml]: please add `license_file: LICENSE.txt`, assuming it is in the source distribution.

Comment (jakirkham) [inline recipes/pandas-compat/meta.yaml]: I think it is not unfortunately.

ref: https://github.com/pandas-dev/pandas-compat/commit/9e6d0d145a635cf280a8da8cc02ab0ffc30f3cde#diff-97c91a104c431d0c365565d3ac03ac13R3

**Comment by jreback** [comment]
ok pushed a new version with the license.

**Comment by jakirkham** [comment]
Anything else on your end @synapticarbors?

**Comment by jreback** [comment]
thanks guys
very efficient!

**Pull request merged**